backend/routes/payments.py

```
@@ -243,6 +243,31 @@ def get_subscription(
     db: Session = Depends(get_db),
 ):
     """Return the current tenant's active subscription (or 404 if on free plan)."""
+    if config.DISABLE_PAYMENTS:
+        # Return a mock Pro subscription if payments are disabled
+        pro_plan = db.query(Plan).filter(Plan.code == "pro").first()
+        if not pro_plan:
+            # Fallback if DB isn't seeded yet
+            pro_plan = Plan(
+                id=uuid.uuid4(),
```

Ruff (0.15.14)

[error] Undefined name `uuid`

(F821)